app-misc/vcontrold/vcontrold-0.98.8.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
You should use the latest available EAPI, which is currently 7.

Reply:
Okay, fixed